apis/pipeline/pipeline.yaml

```diff
@@ -7954,6 +7954,21 @@ components:
                 secretId:
                     type: string
                     example: "61b9707ca463cad79188bb0ddfa1345e41fc01b651feddf06eea58a25c56d717"
+                prefix:
+                    type: string
+                    example: "my-backup-prefix"
+                    description: "Optional, if not provided will be set to the name of the cluster"
+                location:
+                    type: string
+                    example: "us-east"
+                storageAccount:
+                    type: string
+                    example: "my-storage-account"
+                    description: "required only case of Azure"
```

Comment:
**Note**: typo.
```suggestion
                    description: "required only in case of Azure"
```

Similarly at:
https://github.com/banzaicloud/pipeline/pull/3278/files#diff-d41e335500d6bf91169555b68a970b732b21f70dbecef99585ec49d11f7758b7R7971
https://github.com/banzaicloud/pipeline/pull/3278/files#diff-d41e335500d6bf91169555b68a970b732b21f70dbecef99585ec49d11f7758b7R8085
https://github.com/banzaicloud/pipeline/pull/3278/files#diff-d41e335500d6bf91169555b68a970b732b21f70dbecef99585ec49d11f7758b7R8089